public_html/formCreator/templates/formPreview.php

```diff
@@ -141,9 +141,40 @@
                         <div class="row group noHide">
                             <div class="control-group " id="formSettings_formContainer_container">
                                 <ul class="checkboxList">
-                                    <li><label class="checkbox" for="formSettings_formContainer"><input type="checkbox" id="formSettings_formContainer" name="formSettings_formContainer" {local var="formContainer"}> Act as Container</label></li>
-                                    <li><label class="checkbox" for="formSettings_formProduction"><input type="checkbox" id="formSettings_formProduction" name="formSettings_formProduction" {local var="formProduction"}> Production Ready</label></li>
-                                    <li><label class="checkbox" for="formSettings_formMetadata"><input type="checkbox" id="formSettings_formMetadata" name="formSettings_formMetadata" {local var="formMetadata"}> Metadata Form</label></li>
+                                    <li>
+                                      <label class="checkbox" for="formSettings_formContainer">
+                                        <input type="checkbox" id="formSettings_formContainer" name="formSettings_formContainer" {local var="formContainer"}>
+                                        Act as Container
```

Comment:
Why did you change the formatting here? It makes it harder to see if you changed anything in the code and reduces the usefulness of git blame.